src/Uno.UWPSyncGenerator/NamedTypeSymbolStringComparer.cs

```diff
@@ -13,7 +13,7 @@ public bool Equals(INamedTypeSymbol x, INamedTypeSymbol y)
 
 		public int GetHashCode(INamedTypeSymbol obj)
 		{
-			return obj.GetHashCode();
```

Comment:
This may not be valid. Two types with the same `DisplayName` may really be different if located in different assemblies, as they're not fully qualified.

Which problem did you identify that this change fixes?

Reply:
@jeromelaban See the `Equals` method implementation. This equality comparer is specifically for display name equality (hence the `StringComparer` in the name). Anything I'm missing?

Reply:
Indeed, I missed that part. Let use your change then!